deploy.sh

```diff
@@ -60,6 +60,6 @@ mvn --settings $OSSRH_DEPLOY_SETTINGS_XML -DskipTests clean deploy
 #confluent release
 mvn -f pom_confluent.xml --settings $OSSRH_DEPLOY_SETTINGS_XML -DskipTests clean package
 #white source
-whitesource/run_whitesource.sh
+# whitesource/run_whitesource.sh
```

Comment:
Instead of comment, just remove it? unless we are planning to use in future? also delete the whitesource.sh script?

Reply:
Lets Remove this with next release.